README.md

```diff
@@ -46,8 +46,15 @@ A command is required
 npm install @usedispatch/client
 ```
 
+```bash
+npm install @usedispatch/forum
+```
+
+To learn more about using forums, check out our [forums react SDK](./forums_react.md).
 ### Quickstart
 
+#### Messaging Example
+
 ```typescript
 
 import * as web3 from '@solana/web3.js';
```

Comment:
dispatch cli is quite out of date right now, not sure if it even works. lets pull it out of the main page and make it into its own section (and could you please install and verify that it works? if not, we'll have to file a ticket to fix it, and remove it from here till it works)